CHANGELOG.md

```diff
@@ -1,7 +1,17 @@
-## master
-    - Support for chunk cache configuration at data set create/open
-    - Support for std::array (unnested)
+## Version 2.1 - 2019/10/XX
+### API:
+    - DataType class gained getClass(), getSize() and string()
+    - Object gained getInfo() and getType()
+    - Node_Traits gained getLinkType() and getObjectType()
+    - Node_Traits::createGroup() gained the parents argument to specify if intermediates groups should be created too
     - Added AnnotateTraits::createAttribute and NodeTraits::createDataset overloads to create and initialize the data in a single call
+    - Add the new H5Easy high level API.
+
+### FEATURE:
+    - Add a lot of new platform supported on travis (xcode, windows, clang, ...)
+    - DataSet now accept std::array and std::complex type
+    - Support for chunk cache configuration at data set create/open
+    - Add doc target to cmake and publish doc on: https://bluebrain.github.io/HighFive/
```

Comment:
We should also tag these entries with the related issues.

Reply:
Add a section Bug Fixes

Reply:
Can you create yourself the bug fixes section.
In fact, I tried, but it was not easy for me (I didn't know the library enough).